helix-term/src/ui/spinner.rs

```diff
@@ -21,9 +29,9 @@ impl Default for Spinner {
     }
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
 pub struct Spinner {
-    frames: Vec<&'static str>,
+    frames: Vec<char>,
```

Comment:
This will limit the possible characters afaik. Some unicode is more than one char. This might not a problem though.

Reply:
You may be right, but I was under the impression that a `char` was a unicode code point - so you should be able to use a single unicode code point for each frame, regardless of the number of ("traditional char") bytes.

Reply:
It is. I'm talking about grapheme clusters, that may be multiple unicode points.

Don't you love unicode sometimes 😉

Reply:
LOL. I hate unicode all the time.

I'm inclined to go back to `Vec<&'static str>` and leave this as an exercise for someone else :)

Reply:
On second thought - I can't find a good way to get `Vec<String>` or `Vec<&'static str>` without running into borrow checker madness. I think if we say, "you can't use grapheme clusters in your custom spinner" it's reasonable, especially since right now you can't customize it at all. What do you think?

Reply:
Vec<'static str> can't be loaded at run-time without a memory-leak so that won't work.

Vec<String> should work just fine however.

Reply:
Vec of what? `String`?

Reply:
It must have disappeared somehow. Yes String

Reply:
Moved to `Vec<String>`.